src/Bridge/Symfony/Serializer/Normalizer/EnumNormalizer.php

```diff
@@ -27,8 +27,10 @@ class EnumNormalizer implements NormalizerInterface, DenormalizerInterface
      * {@inheritdoc}
      *
      * @param EnumInterface $object
+     *
+     * @return string|int
```

Comment:
```suggestion
     * @return mixed
```

Peut être voir si on peux mettre des générics phpstan ici ( en mettant le param EnumInterface<T> et return T)

Reply:
I don't think it'll works here, since we have no way to indicate the template type.
It works when extending a base class defining a template. I'm not sure it's valid as is.